source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACOHostConfig.mm

```diff
@@ -19,15 +20,17 @@ @implementation ACOHostConfig
 
 - (instancetype)init
 {
-    self = [super init];
+    self = [self initWithConfig:nil];
```

Comment:
>nil [](start = 32, length = 3)

nullptr

Reply:
thanks

---
In reply to: [171736222](https://github.com/Microsoft/AdaptiveCards/pull/1178#discussion_r171736222) [](ancestors = 171736222)